packages/blocks/google-cloud/src/lib/actions/custom-google-cloud-api-action.ts

```diff
@@ -0,0 +1,33 @@
+import { createCustomApiCallAction } from '@openops/blocks-common';
+import { googleCloudAuth } from '@openops/common';
+import { runCommands } from '../google-cloud-cli';
+
+async function getAccessToken(auth: any): Promise<string> {
```

Comment:
there are other ways to do this, but this is the easiest that does not require a change to our current auth, while also using our existing structure

Reply:
nope, also forgot to add projects dropdown, same with azure